pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -5,6 +5,28 @@ if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" != 1 ] && [ "${LIMA_CIDATA_CONTAINERD_US
 	exit 0
 fi
 
+if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" = 1 ] && [ -f /etc/alpine-release ]; then
+	# need to use the containerd packages
```

Comment:
Please explain the reason (glibc) in the comments

Reply:
Can explain that containerd is dynamic and buildkitd is static